pyproject.toml

```diff
@@ -21,7 +21,6 @@ dependencies = [
 
 [project.optional-dependencies]
 test = [
-    "black[jupyter]",
```

Comment:
I'm new to ruff, but it looks like we can bring in [formatting for notebooks](https://docs.astral.sh/ruff/faq/#does-ruff-support-jupyter-notebooks) with:

```
[tool.ruff]
extend-include = ["*.ipynb"]
```